drivers/CMakeLists.txt

```diff
@@ -1,30 +1,33 @@
-add_subdirectory_if_kconfig(adc)
-add_subdirectory_ifdef(CONFIG_AIO_COMPARATOR    aio)
 add_subdirectory(console)
+add_subdirectory(crc)
+add_subdirectory(display)
+add_subdirectory(interrupt_controller)
+
+add_subdirectory_if_kconfig(adc)
 add_subdirectory_if_kconfig(bluetooth)
 add_subdirectory_if_kconfig(clock_control)
 add_subdirectory_if_kconfig(counter)
-add_subdirectory(crc)
-add_subdirectory(display)
 add_subdirectory_if_kconfig(dma)
-add_subdirectory_ifdef(CONFIG_NET_L2_ETHERNET   ethernet)
 add_subdirectory_if_kconfig(flash)
 add_subdirectory_if_kconfig(gpio)
 add_subdirectory_if_kconfig(grove)
 add_subdirectory_if_kconfig(i2c)
 add_subdirectory_if_kconfig(ieee802154)
-add_subdirectory(interrupt_controller)
 add_subdirectory_if_kconfig(ipm)
+add_subdirectory_if_kconfig(led_strip)
+add_subdirectory_if_kconfig(net)
```

Comment:
I do not think we have CONFIG_NET, so this should be plain add_subdirectory(net)

Reply:
Good catch. This was a mistake.

Reply:
I ifdef'd on CONFIG_NETWORKING to align with KBuild.